packages/server/src/modules/step/__tests__/get-step-route.spec.ts

```diff
@@ -0,0 +1,14 @@
+import request from 'supertest';
+
+import application from '@application';
+
+describe('GET /step/:stepId', () => {
```

Comment:
Let's use this opportunity to do the rest of the setup.

- [ ] Define a way for mocking a token
- [ ] Create DB container for testing
- [ ] Create a separate env file for tests

@brama051 do you have anything else that was blocking you to add more tests?